forwards_paper.tex

```diff
@@ -240,10 +241,50 @@ \subsection*{Simulation benchmarks}
 
 \plr{Insert figures/tables of performance here and describe them.}
 
+The total run times for simulations with selection are shown in Figure~\ref{fig:runtimes_selection}.  When tracking
+neutral mutations (instead of the pedigree), run times increase dramatically with increasing region size ($4Nr = 4Nu$ for
+these simulations).  With \fwdpy{}, simulations with $N=5 \times 10^4$ timed out for region sizes larger than $10^3$.
+With \simupop{}, we were only able to run simulations with $N=10^3$ and region sizes up to $10^4$.  Pedigree tracking
+dramatically improves run times for both \fwdpy{} and \simupop{} (right column of Figure~\ref{fig:runtimes_selection}).  For parameters that ran to completion when tracking neutral mutations and when tracking
+the pedigree, the relative speedup due to pedigree tracking is up to $\approx 50$ fold (Figure~\ref{fig:relative_speedup_selection}).  For \fwdpy{}, pedigree tracking reduces runtimes more for larger population sizes
+(Figure~\ref{fig:relative_speedup_selection}).  We could not make such an observation with \simupop{}, which tended to
+run out of memory on our system.  Note the log scale of the x axis in Figure~\ref{fig:runtimes_selection}, which
```

Comment:
Good dry tone with 'tended to run out of memory'.

We should note the 50x above only applies there and the common scale obscures the speedup with simupop which is about 5x. we could leave the same scale and note the average speedup for simupop increases slowly from $\approx 4$ to $\approx 5$ over the range shown
```
    sel.x   rho ave_speedup
   <fctr> <dbl>       <dbl>
 1  False  1000    4.097328
 2  False  2500    4.080286
 3  False  5000    4.353546
 4  False  7500    5.695227
 5  False 10000    5.723364
 6   True  1000    4.325214
 7   True  2500    4.126641
 8   True  5000    4.860460
 9   True  7500    4.562465
10   True 10000    5.054865

````

Reply:
![image](https://user-images.githubusercontent.com/510776/34310898-dafcd3c0-e70e-11e7-9457-12c68db9a26d.png)

Reply:
Perhaps should remove the common scale for speedup...